chipsec_tools/compression/build.cmd

```diff
@@ -16,27 +16,30 @@
 @REM #Contact information:
 @REM #chipsec@intel.com
 
+goto :main
+
+:set_buildvars
+@echo Building compression
+@echo off
+@REM # Need to set EDK_TOOLS_PATH and BASE_TOOLS_PATH to current directory and HOST_ARCH
```

Comment:
Does HOST_ARCH still need to be set?

Reply:
should be 32 bit to compile, will remove the comment